jobs/wf2-5-createEvents.js

```diff
@@ -0,0 +1,67 @@
+fn(state => {
+  const TEIs = {};
+  return { ...state, TEIs };
+});
+
+// Fetch TEI's for each patient
+each('encounters[*]', state => {
+  const encounter = state.data;
+
+  return get(
```

Comment:
This `get` can be restructured like `each('encounters[*]', get('...', (state) => { out, filter })`

You will still need to use the callback function.

I am happy to pair with you on this - just give me a shout when you start!